### PR TITLE
Add to changelog for SDK version 5.1.0

### PR DESCRIPTION
**Pull request by DorothySun216**

### CODE REVIEW

File: sdk/servicebus/Microsoft.Azure.ServiceBus/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Release History
 ## 5.1.0-beta.1 (Unreleased)
 
+## 5.1.0 (2020-11-20)
```
Comment (JoshLove-msft) [inline sdk/servicebus/Microsoft.Azure.ServiceBus/CHANGELOG.md:4]: Why are we bumping the minor version for bug fixes. I would expect this to be 5.0.1.

Comment (DorothySun216) [reply sdk/servicebus/Microsoft.Azure.ServiceBus/CHANGELOG.md:4]: Let me double check with @yvgopal about whether his fix is an improvement or not. From the name I think it is a bug fix.

Comment (yvgopal) [reply sdk/servicebus/Microsoft.Azure.ServiceBus/CHANGELOG.md:4]: atom-xml fix is more than a bug fix. So I suggested to change the version to 5.1.0

---
In reply to: [527999644](https://github.com/Azure/azure-sdk-for-net/pull/17131#discussion_r527999644) [](ancestors = 527999644)

**Review by yvgopal — APPROVED** [review]
:shipit:

**Pull request merged**